esp32-hal/build.rs

```diff
@@ -35,6 +35,9 @@ fn main() {
     // Only re-run the build script when memory.x is changed,
     // instead of when any part of the source code changes.
     println!("cargo:rerun-if-changed=ld/memory.x");
+
+    #[cfg(feature = "defmt")]
+    println!("cargo:rustc-link-arg=-Tdefmt.x");
```

Comment:
Are we not able to just do this in `esp-hal-common/build.rs` once, so we don't have to copy-paste this into each package's build script?

Reply:
This doesn't seem to work.